app/controllers/not_found_controller.rb

```diff
@@ -0,0 +1,5 @@
+class NotFoundController < ApplicationController
+  def not_found
```

Comment:
Nit: We can move this to the errors controller and use the application layout just for this action:

```ruby
class ErrorsController < ActionController::Base
  ...
  def not_found
     render status: :not_found, layout: 'application'
  end
```

Main benefit would be having all error actions in the one controller so they easier to find and extend.

Reply:
> Nit: We can move this to the errors controller and use the application layout just for this action:

We can't, or at least not easily; the application layout intrinsically depends on the methods of ApplicationController, and just setting it here just results in a bunch of internal server errors and unknown/missing methods

Reply:
Ahh ok, sorry, when testing locally I forgot to switch the route over lol

In that case, I think we're still best to aim for consistency with these by losing the navbar and footer for the 404 page and just putting it all through the errors layout.

We could maybe make the link back to home clearer but other than that it doesn't look bad:
<img width="1693" alt="Screenshot 2023-07-03 at 06 15 46" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/a561241d-0f3a-4a4a-939b-b805302ce9ca">

Another option would be giving the same kind of styling as 500/422 pages so they are completely consistent. But using the current styling isn't a deal breaker.